packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/types.ts

```diff
@@ -16,19 +23,55 @@ export type FastLinkType = {
     tokenExpiresAt: string
   }
   id: string
-  partner: 'YODLEE'
+  partner: BankPartners.YODLEE
 }
 
-export type BankStatusType =
-  | 'ACTIVE'
-  | 'BANK_TRANSFER_ACCOUNT_INFO_NOT_FOUND'
-  | 'BANK_TRANSFER_ACCOUNT_ALREADY_LINKED'
-  | 'BANK_TRANSFER_ACCOUNT_NAME_MISMATCH'
-  | 'DEFAULT_ERROR'
+export type OBType = {
+  attributes: OBAttributesType
+  id: string
+  partner: BankPartners.YAPILY
+}
+
+interface OBCountryType {
+  countryCode2: string
+  displayName: string
+}
+
+interface OBMediaType {
+  source: string
+  type: 'string'
+}
+export interface OBInstitution {
+  countries: OBCountryType[]
+  credentialsType: string
+  environmentType: string
+  features: [string]
```

Comment:
just curious any specific reason why we need tuple there?

Reply:
good spot. that's just a typo. Should just be an array of strings, not a tuple.